### PR TITLE
reworded claims to be a bit happier

### PR DESCRIPTION
**Pull request by gkiar**

### CODE REVIEW

**Comment by gkiar** [comment]
cc: @glatard

File: paper/impact-of-instability.tex [hunk 1]
```diff
@@ -61,7 +61,7 @@
 \JournalInfo{$ $ } % Journal information
 \Archive{Pre-print} % Additional notes (e.g. copyright, DOI, review/research article)
 
-\PaperTitle{Numerical Instabilities in Analytical Pipelines Compromise the Reliability of Network Neuroscience} % Article title
+\PaperTitle{Numerical Instabilities Lead to Meaningful Variability in Brain Networks} % Article title
```
Comment (glatard) [inline paper/impact-of-instability.tex]: How about 'Large and Meaningful', to suggest both the positive and negative aspects of variability?

Comment (glatard) [reply paper/impact-of-instability.tex]: I would vote for 'Numerical Instabilities in Analytical Pipelines Lead to Large and Meaningful Variability in Brain Networks'

Comment (gkiar) [reply paper/impact-of-instability.tex]: 👍

**Pull request merged**